packages/roosterjs-content-model-plugins/lib/touch/TouchPlugin.ts

```diff
@@ -69,70 +71,14 @@ export class TouchPlugin implements EditorPlugin {
                 this.timer = targetWindow.setTimeout(() => {
                     this.timer = 0;
 
-                    if (!this.isDblClicked && this.editor) {
-                        const caretPosition = getNodePositionFromEvent(
-                            this.editor,
-                            event.rawEvent.x,
-                            event.rawEvent.y
-                        );
+                    if (this.editor) {
+                        if (!this.isDblClicked) {
```

Comment:
I remember in MouseEvent there is a property to tell you how many times it is clicked as a single event which can be used by detecting double click, triple click, ... Not sure if that also works in PointerEvent, can you double check?

Reply:
I see. Ok.